src/smart-components/workspaces/create-workspace/CreateWorkspaceWizard.tsx

```diff
@@ -52,4 +58,20 @@ export const CreateWorkspaceWizard: React.FunctionComponent<CreateWorkspaceWizar
   );
 };
 
-export default CreateWorkspaceWizard;
+const CreateWorkspaceWizardWrapper: React.FC<CreateWorkspaceWizardProps> = ({ ...props }) => {
+  return (
+    <IntlProvider locale={locale} messages={messages[locale]}>
+      <RegistryContext.Provider
+        value={{
+          getRegistry: () => registry,
+        }}
+      >
+        <Provider store={registry.getStore()}>
+          <CreateWorkspaceWizard {...props} />
+        </Provider>
+      </RegistryContext.Provider>
+    </IntlProvider>
+  );
+};
+
+export default CreateWorkspaceWizardWrapper;
```

Comment:
Let's move this into seperate file `CreateWorkspaceWizardModule` this way we don't have to change previous exposed component and have full control over what is being used outside RBAC.